test/acceptance/features/examples.feature

```diff
@@ -7,6 +7,7 @@ Feature: Verify examples provided in Service Binding Operator github repository
     Background:
         Given Namespace [TEST_NAMESPACE] is used
         * Service Binding Operator is running
+        * PostgreSQL DB operator is installed
```

Comment:
this would mean that each scenario need postgresql operator, but it does not. Hence, this step should be moved into the scenario.

Reply:
There are upcoming examples which need postgresql operator. One of them is PR #768.